packages/grid/_modules_/grid/models/gridExport.ts

```diff
@@ -1,12 +1,12 @@
 /**
- * Available export formats. To be extended in future.
+ * The options to apply on the CSV export.
  */
-export type GridExportFormat = 'csv';
+export interface GridExportCsvOptions {
+  fileName?: string;
+  utf8WithBom?: boolean;
```

Comment:
do we need the `withBom` suffix? I don't think it brings any value

Reply:
@dtassone I don't understand. How will a developer differentiate if there is a BOM or not?

Reply:
what they want is UTF8

Reply:
I don't follow. Isn't it already UTF8?

Reply:
Not sure actually, I thought it was ASCII and we were missing the special characters.

Reply:
How about we start without, only an option to customize the filename?
```suggestion
```

Reply:
The main reason for this pull request was to add an option for developers to set a flag if they wanted to add a BOM for a utf-8 encoded file as mentioned in #1440. The filename setting came up by the way. What's the problem with the utf8WithBom option? If you really don't want this option, we can remove it.

Reply:
I don't think it's a good idea to add a BOM by default. According to the Unicode Standard: 

> Use of a BOM is neither required nor recommended for UTF-8, but may be encountered in contexts where UTF-8 data is converted from other encoding forms that use a BOM or where the BOM is used as a UTF-8 signature

[https://en.wikipedia.org/wiki/Byte_order_mark](url)

Reply:
More context: https://github.com/ag-grid/ag-grid/issues/3916